dao-contracts/src/voting/governance_voting/voting.rs

```diff
@@ -119,15 +119,15 @@ impl Voting {
         }
     }
 
-    pub fn get_quorum(&self) -> usize {
+    pub fn get_quorum(&self) -> U256 {
         match self.get_voting_type() {
-            VotingType::Informal => self.informal_voting_quorum.as_usize(),
-            VotingType::Formal => self.formal_voting_quorum.as_usize(),
+            VotingType::Informal => self.informal_voting_quorum,
+            VotingType::Formal => self.formal_voting_quorum,
         }
     }
 
-    pub fn get_result(&self, voters_number: usize) -> VotingResult {
-        if self.get_quorum() > voters_number {
+    pub fn get_result(&self, voters_number: u32) -> VotingResult {
```

Comment:
Bit weird that `get_quorum` returns `U256` but `votes_number` is `u32`.

Reply:
It simplifies the code. Quorum is connected to data saved in storage, thus U256. `voters_number` is just a len of a mapping vector.

Reply:
I think that setting types to more realistic values can be done at the end.

Reply:
So reduce quorums in the variable repo to `u32`, as it doesn't make sens to have it as `U256`.